schemas/src/registry_schemas/schemas/incorporation.json

```diff
@@ -0,0 +1,50 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://bcrs.gov.bc.ca/.well_known/schemas/incorporation",
+    "required": [
+        "incorporation"
+    ],
+    "type": "object",
+    "title": "Incorporation Filing",
+    "properties": {
+        "incorporation": {
+            "type": "object",
+            "required": [
+                "nameRequest",
+                "businessContact"
+            ],
+            "properties": {
+                "nameRequest": {
+                    "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/name_request"
+                },
+                "offices": {
+                    "registeredOffice": {
+                        "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/office"
+                    },
+                    "recordsOffice": {
+                        "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/office"
+                    },
+                    "required": [
+                        "registeredOffice"
+                    ]
+                },
+                "businessContact" : {
```

Comment:
might want to make this a sub-schema for use in businessContact and NR submitter/applicant. See notes below.

Reply:
Good idea, we can do that for sure, thanks!